includes/api/class-give-api.php

```diff
@@ -1037,6 +1037,13 @@ private function get_form_data( $form_info ) {
 			$form['info']['tags'] = get_the_terms( $form_info, 'give_forms_tag' );
 		}
 
+		if ( give_is_setting_enabled( give_get_meta( $form_info->ID, '_give_goal_option', true ) ) ) {
+			$goal_amount = give_get_meta( $form_info->ID, '_give_set_goal', true );
+			$goal_percentage_completed = round( ( give_get_form_earnings_stats( $form_info->ID ) / $goal_amount ) * 100 );
```

Comment:
@mehul0810 This code will generate PHP notice.
```
PHP Warning:  Division by zero in /wp-content/plugins/Give/includes/api/class-give-api.php on line 1042
```

And also max percentage should be `100`
for ref: https://github.com/WordImpress/Give/blob/master/templates/shortcode-goal.php#L50